getting-started.md

```diff
@@ -42,6 +42,17 @@ node createUser.js
 
  This will prompt you to set a username and password, and then create the entries in the mongo database. The password is hashed.
 
+## Edit the initRegistry.js file
+
+Inside initRegistry.js change the username and password to your own
+
+```bash
```

Comment:
use Javascript instead of bash.

Reply:
I have made the changes

Reply:
Perfect!

Reply:
Thank you @ankitskvmdam  for the opportunity, this has been my first open-source contribution.